### DISCUSSION TITLE
How do I delete or disable a pipeline?

### DISCUSSION THREAD

**Discussion by govardha** [signalfx/splunk-otel-collector-chart#624] [Q&A] [answered]
Hello,

Is it possible for me to disable a built in pipeline that the helm chart gives completely?  I tried to set the pipeline to null but otel complains it at least needs one receiver.  

Thank you!
G

**Comment by atoulme** [comment, marked as answer]
That would likely require some customization of the helm chart. Please feel free to open an issue if you have a specific need.
